haarpy/tests/test_symmetric.py

```diff
@@ -285,6 +285,196 @@ def test_irrep_dimension_murn_naka_rule(partition):
     assert ap.irrep_dimension(partition) == ap.murn_naka_rule(partition, conjugacy_identity)
 
 
+@pytest.mark.parametrize("degree", range(1,8))
+def test_sorting_permutation_single_len(degree):
+    "Test that the size of the permutation is the size of the sorted sequence"
```

Comment:
In this an all the test that follow you should use triple quotation marks to be consistent with the style in the rest of the library.